### ISSUE TITLE
hb.keras.Model's fit() func support dataset multiple labels

### ISSUE THREAD

**Issue by karterotte** [alibaba/HybridBackend#85]
# User Story

I want to train model using multiple labels data, but fit function throw Exception like
`Error when checking model target:expected no data....`

When the dataset only contain one label, It's OK.

# Detailed requirements

- The dataset like this,the labels maybe tuple or dict:
```
ds = hb.data.ParquetDataset(XXX)
def map_fn(batch):
    labels = tuple([batch[l] for l in labels])
    features = {}
    #pass
    return features, labels
ds = ds.map(map_fn)
```
- The fit() like this:
```
m.fit(
    x=train_ds,
    validation_data=valid_ds,
    #XXX
    verbose=0)
```

I wish fit() can support dataset like above.

# API Compatibility
hb.keras.Model's fit()
# Willing to contribute

Yes
